apps/extension/src/features/accounts/account-card.tsx

```diff
@@ -0,0 +1,376 @@
+                {account.isPrimary && (
+                  <Tooltip.Root delayDuration={300}>
+                    <Tooltip.Trigger asChild>
+                      <div className="px-2 py-1 text-xs text-white rounded bg-blue-9 cursor-help">
+                        Organizer
+                      </div>
+                    </Tooltip.Trigger>
+                    <Tooltip.Content
+                      align="end"
+                      side="right"
+                      sideOffset={10}
+                      className="z-10"
+                    >
+                      <div className="px-5 py-6 bg-white rounded-lg shadow-2xl w-[350]">
+                        <div className="font-semibold">Organizer Account</div>
+                        <div className="pt-2">
+                          The selected account will be the organizer for all
+                          events created.
+                        </div>
+                      </div>
+                    </Tooltip.Content>
+                  </Tooltip.Root>
+                )}
+              </div>
+              <RadioGroup.Root asChild defaultValue={account?.eventColor}>
+                <div className="flex pt-4 space-x-4">
+                  {colors?.map((color, ix) => {
+                    // const isSelected = account.color.id === id;
+
+                    return (
+                      <RadioGroup.Item
+                        key={ix}
+                        value={color}
+                        className="relative w-[2.25rem] h-[2.25rem] border rounded cursor-pointer"
+                        onClick={() =>
+                          updateEventColor({
+                            id: account.id,
+                            eventColor: color,
+                          })
+                        }
+                        style={{
+                          backgroundColor: themeColors[color][7],
+                          borderColor: themeColors[color][11],
+                        }}
+                      >
+                        <RadioGroup.Indicator asChild>
+                          <div className="absolute bottom-[2px] right-[2px] flex">
+                            <span className="inline-block w-3.5 h-3.5 -rotate-[140deg]">
+                              <div
+                                id="circle"
+                                className="absolute w-3.5 h-3.5 rounded-xl  left-[1px] top-[1px]"
+                                style={{
+                                  backgroundColor: themeColors[color][11],
+                                }}
+                              ></div>
+                              <div
+                                id="checkmark-stem"
+                                className={clsx(
+                                  [
+                                    "w-[1px] h-[8px] absolute top-[5px] left-1.5",
+                                  ],
+                                  `bg-${color}-9`,
```

Comment:
tailwind's usually not happy when composing classnames like this, you set the bg with styles prop so this can be omitted right?

Reply:
Yes, it's outdated and I'll remove.

Using inline style to follow the [jit mode guide](https://v2.tailwindcss.com/docs/just-in-time-mode) (even though it's commented out right now 😅 )
<img width="847" alt="CleanShot 2022-11-23 at 09 08 59@2x" src="https://user-images.githubusercontent.com/18185649/203497905-d6e9c197-8135-44b5-acd1-9d36b7a587fe.png">

Reply:
Removed